lib/ex_doc/autolink.ex

```diff
@@ -85,6 +88,12 @@ defmodule ExDoc.Autolink do
     {tag, attrs, walk(ast, config), meta}
   end
 
+  defp remove_link({:a, _attrs, inner, _meta}),
+    do: inner
+
+  defp remove_link(ast),
+    do: ast
```

Comment:
this clause is not needed, please remove it or add a test that exercises it :)

Reply:
Agreed with that is unneeded, but how would you go about testing it a private function that is called by private functions.

Reply:
> I wouldn't test a private function!
> 
> I think with our autolink tests we strike a nice enough balance, the tests are sufficiently low level (in that we don't need to spin up the entire exdoc to test something) but at the same time high-level enough that we test the external behaviour, what was the returned doc AST (and potential side effects like warnings), we don't test implementation details like what got called with what, etc.

Sorry for the confusion, when I read your message I read "please remove it AND add a test that exercises it!", that' s why I was a bit puzzled.
I haven't had my morning coffee yet, that's probably it :smile: